frappe/public/js/frappe/form/quick_entry.js

```diff
@@ -1,129 +1,178 @@
 frappe.provide('frappe.ui.form');
 
-frappe.ui.form.quick_entry = function(doctype, success) {
-	frappe.model.with_doctype(doctype, function() {
-		var mandatory = $.map(frappe.get_meta(doctype).fields,
-			function(d) { return (d.reqd || d.bold && !d.read_only) ? d : null });
-		var meta = frappe.get_meta(doctype);
-
-		var doc = frappe.model.get_new_doc(doctype, null, null, true);
+frappe.ui.form.QuickEntryController = Class.extend({
```

Comment:
just call it `frappe.ui.form.QuickEntryForm` 

Its not a controller but a form